docs/rfcs/91-enhance-ls-command.md

```diff
@@ -0,0 +1,240 @@
+- Author: JinnyYi <github.com/JinnyYi>
+- Start Date: 2021-11-02
+- RFC PR: [beyondstorage/beyond-ctl#91](https://github.com/beyondstorage/beyond-ctl/issues/91)
+- Tracking Issue: [beyondstorage/beyond-ctl#0](https://github.com/beyondstorage/beyond-ctl/issues/0)
+
+# BCP-91: Enhance ls Command

```

Comment:
How about `Add Summerize Support in ls`? More specified to me.

Reply:
OK. 

Does `summerize` need to count the files in all subdirectories? If only the current directory is counted, do we need to support recursively traverse a directory?

Reply:
I think this is why `-r` exists. If the user uses `-r` & `--summerize` at the same time, we should recursively traverse it, otherwise, we don't need to.

Reply:
Got it.